package.json

```diff
@@ -33,11 +33,11 @@
   "homepage": "https://github.com/geostyler/geostyler-legend#readme",
   "dependencies": {
     "@types/d3-selection": "^3.0.1",
-    "geostyler-openlayers-parser": "^3.0.0"
+    "geostyler-openlayers-parser": "^4.0.0"
   },
   "peerDependencies": {
-    "d3": "^6.7.0",
-    "ol": "6 - 7"
+    "d3": ">=6",
+    "ol": ">=7"
```

Comment:
Is there a reason that `geostyler` supports versions 6 and above, but here we limit the range to 7 and above? See https://github.com/geostyler/geostyler/pull/1962

Reply:
My bad. This should be `>=6`, see [70b498e](https://github.com/geostyler/geostyler-legend/pull/416/commits/70b498ebf6b25705740bd9697432ebf63b799587).